scholarshipr/Scholarship.js

```diff
@@ -12,7 +12,7 @@ var scholarshipSchema = new Schema({
     description: {type: String, required: false, unique: false},
     dollarAmount: {type: Number, required: false, unique: false},
     // gender: {type: Enumerator, required: false, unique: false},
-    approvalStatus: {type: Boolean, required: true, unique: false},
+    approvalStatus: {type: String, required: true, unique: false},
```

Comment:
I think we should keep _approvalStatus_ as a boolean for future suggested scholarship features and usability by HTML toggles.

Reply:
On line 32 of index.js we have some code that handles this I think. I'm not totally sure which way of handling it is best but it seems to currently be working to me.